client/customer.go

```diff
@@ -17,6 +17,18 @@ func (c *Client) ListCustomers(appID string, appType string) ([]types.Customer,
 	return nil, errors.Errorf("unknown app type %q", appType)
 }
 
+// list customers by app and app version
+func (c *Client) ListCustomersByAppAndVersion(appID string, appVersion string, appType string) ([]types.Customer, error) {
+
+	if appType == "platform" {
+		return nil, errors.New("listing customers is not supported for platform applications")
```

Comment:
`listing customers by app version is not supported for native scheduler applications`

Reply:
Changed text per suggestion. "listing customers by app version is not supported for native scheduler applications"